src/server/api/v1/controllers/variableApiController.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const variableManager = require("../../../../backend//variables/replace-variable-manager");
+
+exports.getReplaceVariables = function(req, res) {
+    const variables = variableManager.getReplaceVariables();
+
+    if (Array.isArray(variables)) {
+        variables.sort((a, b) => {
+            const handleA = a.definition.handle.toLowerCase();
+            const handleB = b.definition.handle.toLowerCase();
+            if (handleA < handleB) {
+                return -1;
+            }
+            if (handleA > handleB) {
+                return 1;
+            }
+            return 0;
+        });
+    }
+
+    res.json(variables);
+};
```

Comment:
Couple notes on this bad boy:
- `variableManager.getReplaceVariables()` always returns an array so i dont believe there is a need for an Array.isArray check
- we can utilize the built-in string method `localeCompare` for sorting here to shorten our logic
- i dont think forcing lowercase afford us anything real here
- since sort modifies the array in place, its generally good practice to spread the original array into a new one to avoid any unintentional side effects

Altogether, this could be simplified to something like:
```js
exports.getReplaceVariables = function(req, res) {
    const sortedVariables = [...manager.getReplaceVariables()]
        .sort((a, b) => a.definition.handle.localeCompare(b.definition.handle));

    res.json(sortedVariables);
};
```